applications/services/cli/cli_commands.c

```diff
@@ -525,6 +658,7 @@ void cli_commands_init(Cli* cli) {
 
     cli_add_command(cli, "vibro", CliCommandFlagDefault, cli_command_vibro, NULL);
     cli_add_command(cli, "led", CliCommandFlagDefault, cli_command_led, NULL);
+    cli_add_command(cli, "buzzer", CliCommandFlagDefault, cli_command_buzzer, NULL);
```

Comment:
We kinda low on free flash memory, adding this feature directly into firmware looks kinda bad.
There is a chance that we will postpone this PR till @portasynthinca3 finish cli refactoring, so we can move non essential things into external apps.